src/makeRobot.js

```diff
@@ -39,6 +39,69 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robotInfo = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
If the key and value have the same name, then it can be written as one word

```suggestion
    name,
    wheels,
    version,
```